crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -276,6 +277,48 @@ impl SequencerBlockData {
         );
         Ok(())
     }
+
+    /// Get the height of the block.
+    #[must_use]
+    pub fn height(&self) -> Height {
+        self.header().height
+    }
+
+    /// Get the height of the block's parent.
+    ///
+    /// # Panics
+    ///
+    /// This function will panic if the block height is less than or equal to 1.
+    /// Only the genesis block has a height of 1, and all other blocks must have
+    /// a larger height.
+    #[must_use]
+    pub fn parent_height(&self) -> Height {
+        assert!(
+            self.height().value() > 0,
+            "block height must be greater than 0"
+        );
+        Height::try_from(self.header().height.value() - 1)
+            .expect("should have been able to decrement tendermint height")
+    }
+
+    /// Get the height of the block's child, or the next block.
+    #[must_use]
+    pub fn child_height(&self) -> Height {
+        self.height().increment()
+    }
```

Comment:
what's the purpose of this (and parent_height) over just getting the height and adding/subtracing in the caller?